ui/raidboss/raidboss_config.ts

```diff
@@ -1762,7 +1762,7 @@ const templateOptions: OptionsTemplate = {
         de: 'Alarm-Text Anzeigedauer (in Sekunden)',
         fr: 'Durée d\'affichage du texte d\'alarme (secondes)',
         ja: '警報テキスト表示時間の長さ (秒)',
-        cn: '警报级别文字显示持续时间 (秒)',
+        cn: '致命级别文字显示持续时间 (秒)',
```

Comment:
```suggestion
        cn: '致命级文字显示持续时间 (秒)',
```

be consistent with the top one.

also, "文字显示持续时间" might be too long for users. I think it makes sense to just "致命级文本显示时长"